tests/conftest.py

```diff
@@ -20,15 +26,15 @@ def pytest_addoption(parser):
     )
 
 
-def pytest_configure(config):
+def pytest_configure(config: "Config") -> None:
```

Comment:
you can use `Config` unquoted if you remove the `if TYPE_CHECKING` pattern above